core/src/rustic_ai/core/guild/dsl.py

```diff
@@ -115,6 +117,28 @@ def evaluate(self, message: JsonDict, agent_state: JsonDict, guild_state: JsonDi
             return False
 
 
+class CelPredicate(RuntimePredicate):
```

Comment:
You should also update `predicate_type_discriminator` and `RuntimePredicateType`